os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/da_ble_central/ble_tizenrt_da_client.c

```diff
@@ -253,27 +300,28 @@ da_ble_result_type rtw_ble_client_delete_bond(da_ble_client_bd_addr* addr)
     }
 
     T_TIZENRT_DELETE_BOND_PARAM *param = os_mem_alloc(0, sizeof(T_TIZENRT_DELETE_BOND_PARAM));
+    if(param == NULL)
+    {
+        printf("\n[%s] Memory allocation failed", __FUNCTION__);
```

Comment:
How about using `debug_print` instead of `printf`?
Please check the line 298 also.

Reply:
OK.